lib/settings.js

```diff
@@ -0,0 +1,36 @@
+var fs = require('fs');
+var AppDirectory = require('appdirectory');
+var dirs = new AppDirectory('sql-seer');
```

Comment:
I'm sorry I mistyped, I meant to say `lib/context.js` which loads settings from multiple files at startup.